src/resolvers/query/app-config/index.js

```diff
@@ -4,10 +4,10 @@ import config from "config";
  * Get information about how app is configured for the cluster.
  */
 export default async function appConfig() {
-  const vars = {
+  return {
     version: config.get("helm.releaseVersion"),
-    baseDomain: config.get("helm.baseDomain")
+    baseDomain: config.get("helm.baseDomain"),
+    smtpEnabled: config.get("email.enabled"),
+    smtpUrl: config.get("email.smtpUrl")
```

Comment:
```suggestion
    smtpUrlExists: !!config.get("email.smtpUrl")
```
Maybe something like this? Maybe there's a better name.

Reply:
Sure thing